slackminion/__main__.py

```diff
@@ -18,7 +19,7 @@ def sigterm_handler(signum, frame):
     args = p.parse_args()
 
     with open(args.config, 'rb') as f:
-        config = yaml.load(f)
+        config = yaml.load(f, Loader=yaml.Loader)
```

Comment:
Can you change this line to use `yaml.SafeLoader`?

Reply:
done, I made this change but forgot to push it.